src/sql/src/statements.rs

```diff
@@ -222,6 +223,7 @@ pub fn sql_value_to_value(
             parse_string_to_value(column_name, s.to_owned(), data_type)?
         }
         SqlValue::HexStringLiteral(s) => parse_hex_string(s)?,
+        SqlValue::Placeholder(s) => return InvalidSqlValueSnafu { value: s }.fail(),
```

Comment:
`return` is unnecessary.

Reply:
Seems not. The whole match is wrapped in an `Ok` as return value.